File: Anton-and-Danik-734A.py
# https://codeforces.com/problemset/problem/734/A
n = int(input())
results = input()


anton_wins = results.count('A')
danik_wins = results.count('D')

if anton_wins > danik_wins:
    print("Anton")
elif anton_wins < danik_wins:
    print("Danik")
else:
    print("Friendship")

File: Anton-and-Danik-734A.c
// https://codeforces.com/problemset/problem/734/A
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    char results[n + 1];
    scanf("%s", results);

    int anton_wins = 0;
    int danik_wins = 0;

    for (int i = 0; i < n; i++) {
        if (results[i] == 'A') {
            anton_wins++;
        } else if (results[i] == 'D') {
            danik_wins++;
        }
    }

    if (anton_wins > danik_wins) {
        printf("Anton\n");
    } else if (anton_wins < danik_wins) {
        printf("Danik\n");
    } else {
        printf("Friendship\n");
    }

    return 0;
}
